packages/next/server/next-server.ts

```diff
@@ -1,19 +1,33 @@
 import type { Params, Route } from './router'
-import type { CacheFs } from '../shared/lib/utils'
-import type { NextParsedUrlQuery, NextUrlWithParsedQuery } from './request-meta'
+import { CacheFs, execOnce } from '../shared/lib/utils'
```

Comment:
`CacheFs` is type, you might want to separate it?

Reply:
Will do it in another PR!